.github/workflows/delete-test-reports.yml

```diff
@@ -0,0 +1,26 @@
+name: Delete Test Report
```

Comment:
nitpick: either use Test**s** Report as we have in the creation job or keep it as is and adapt the other one.

Reply:
Going for `Delete Tests Report`